admin_guide/cluster_management.rst

```diff
@@ -575,6 +575,20 @@ In addition to our core Citus extension, we also maintain several others:
 * `postgresql-topn <https://github.com/citusdata/postgresql-topn>`_ - Returns the top values in a database according to some criteria. Uses an approximation algorithm to provide fast results with modest compute and memory resources.
 * `postgresql-hll <https://github.com/citusdata/postgresql-hll>`_ - HyperLogLog data structure as a native data type. It's a fixed-size, set-like structure used for distinct value counting with tunable precision.
 
+.. _unsupported_extensions:
+
+Unsupported Extensions
+----------------------
+
+* `auto_explain <https://www.postgresql.org/docs/current/static/auto-explain.html>`_ runs EXPLAIN automatically on slow queries. It helps track down un-optimized queries in large applications, but Citus does not support it.
```

Comment:
If I'm understanding it correctly, it's a protection added by Cloud. Maybe what I ought to do for completeness is load auto_explain on my local cluster with the superuser account and watch how citus dies.

Reply:
That would be helpful - I vaguely remember that the main problem was Citus would crash in combination with auto_explain, at least in some cases. But that might have been before Citus switched to use the Postgres custom scan API (i.e. 6.2+ might be good)

There is a separate question if/how we would allow usage of auto_explain on Citus Cloud, but if we are sure it won't crash, then its a different conversation, vs if we know it crashes (in which case it should be documented as a Citus limitation, not something thats specific to Citus Cloud)